assets/src/edit-story/elements/text/display.js

```diff
@@ -36,7 +36,34 @@ import StoryPropTypes from '../../types';
 import { useTransformHandler } from '../../components/transform';
 import { draftMarkupToContent, generateFontFamily } from './util';
 
-const Element = styled.p`
+const Original = styled.p`
+  margin: 0;
+`;
+
+const Clone = styled.p`
+  margin: 0;
+  transform: translateY(-100%);
+
+  span {
+    background: transparent;
+    ${elementWithFontColor}
+  }
+`;
+
+const HighlightElement = styled.span`
+	${elementFillContent}
+	${elementWithFont}
+	${elementWithBackgroundColor}
+	${elementWithStyle}
+  border-radius: 3px;
+  /* stylelint-disable-next-line property-no-vendor-prefix */
+  -webkit-box-decoration-break: clone;
+  box-decoration-break: clone;
```

Comment:
Surprisingly good support for this, but why isn't it auto-prefixed?

Reply:
Hm, it seems to be auto-prefixed and now actually duplicated?

![Screen Shot 2020-03-23 at 6 23 04 PM](https://user-images.githubusercontent.com/637548/77368781-5ed50580-6d33-11ea-854e-486bec88c8b4.png)

Reply:
I think that I was getting some caching issue and I did notice that, but yeah, this explicit declaration is creating duplicates for this case. Just removing. Thanks 👍